test/simulator/timestamp/src/timestamp_manager.cpp

```diff
@@ -90,20 +92,38 @@ timestamp_manager::trim(std::string str)
 }
 
 /* Parse config string to a config map. */
-void
-timestamp_manager::parse_config(
-  const std::string &config, std::map<std::string, std::string> &config_map)
+int
+timestamp_manager::parse_config(const std::string &config,
+  std::map<std::string, std::string> &config_map, const std::vector<std::string> &supported_ops,
+  const std::vector<std::string> &unsupported_ops)
 {
     std::istringstream conf(config);
     std::string token;
 
+    if (config.empty())
+        return (0);
+
     while (std::getline(conf, token, ',')) {
```

Comment:
Can you add a comment saying what we are doing here, i.e what a config would look like and what would be the result in the map?

Reply:
Updated in [ea8f2e5](https://github.com/wiredtiger/wiredtiger/pull/8372/commits/ea8f2e55b685a53034db58860149deeae7bc6d92)